README.md

```diff
@@ -10,6 +10,22 @@ Bind callback functions to OS signals in C and C++.
 SigFn should work in any system with the standard C library,
 which provides the `<signal.h>` header.
 
+## Building Unit Tests
+```bash
+cd <sigfn_root_dir>
+mkdir build && cd build
+cmake .. -DSIGFN_TESTS=ON
+make
+```
+
+## Running Unit Tests
+```bash
```

Comment:
It should be sufficient to either call `ctest -C Debug` or `make test` from the `build` directory.